### ISSUE TITLE
Backintime wont' accept a new "profile".

### ISSUE THREAD

**Issue by Lukon100** [bit-team/backintime#622]
After installing 16.04, I can't run Backintime.

When I start Backintime, it asks me to either restore a previous configuration, or create a new one.

1. There is no previous configuration.

2. After filling out the form to create my "Main Profile" configuration, I click "OK" and nothing happens. No configuration file is created. 

```
DEBUG: [common/backintime.py:509 arg_parse] Arguments: {'debug': True} | unknownArgs: []

Back In Time
Version: 1.1.12

Back In Time comes with ABSOLUTELY NO WARRANTY.
This is free software, and you are welcome to redistribute it
under certain conditions; type `backintime --license' for details.

DEBUG: [common/backintime.py:576 getConfig] config file: /home/luke/.config/backintime/config
DEBUG: [common/backintime.py:577 getConfig] profiles: ['1']
DEBUG: [common/pluginmanager.py:88 PluginManager.load_plugins] Register plugin path /usr/share/backintime/plugins
DEBUG: [common/pluginmanager.py:104 PluginManager.load_plugins] Add plugin qt4plugin.py
DEBUG: [common/pluginmanager.py:104 PluginManager.load_plugins] Add plugin notifyplugin.py
DEBUG: [common/pluginmanager.py:104 PluginManager.load_plugins] Add plugin userscriptsplugin.py
DEBUG: [common/tools.py:611 keyring_supported] Found appropriate keyring 'keyring.backends.SecretService'
DEBUG: [common/tools.py:611 keyring_supported] Found appropriate keyring 'keyring.backends.SecretService'
DEBUG: [common/tools.py:611 keyring_supported] Found appropriate keyring 'keyring.backends.SecretService'
DEBUG: [common/config.py:375 Config.set_snapshots_path] Check snapshot folder: /media/WHEEZE
DEBUG: [common/config.py:385 Config.set_snapshots_path] Create folder: /media/WHEEZE/backintime/ExistenceUBUNTU/luke/1
Traceback (most recent call last):
File "/usr/share/backintime/common/tools.py", line 189, in make_dirs
os.makedirs( path )
File "/usr/lib/python3.5/os.py", line 231, in makedirs
makedirs(head, mode, exist_ok)
File "/usr/lib/python3.5/os.py", line 231, in makedirs
makedirs(head, mode, exist_ok)
File "/usr/lib/python3.5/os.py", line 231, in makedirs
makedirs(head, mode, exist_ok)
File "/usr/lib/python3.5/os.py", line 241, in makedirs
mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/media/WHEEZE/backintime'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "/usr/share/backintime/qt4/settingsdialog.py", line 1698, in accept
if self.validate():
File "/usr/share/backintime/qt4/settingsdialog.py", line 1452, in validate
if not self.save_profile():
File "/usr/share/backintime/qt4/settingsdialog.py", line 1351, in save_profile
ret = self.config.set_snapshots_path( snapshots_path, mode = mode )
File "/usr/share/backintime/common/config.py", line 386, in set_snapshots_path
tools.make_dirs( full_path )
File "/usr/share/backintime/common/tools.py", line 192, in make_dirs
%(path, str(e)), self, 1)
NameError: name 'self' is not defined
DEBUG: [common/configfile.py:476 Config.set_current_profile] change current profile: 1
luke@ExistenceUBUNTU:~$
```

In 14.04 Backintime use to simply work just fine.

I'm trying to set it up to backup certain folders on a mounted NTSF drive called "SLURP" (all my user files) to another mounted NTFS drive called "WHEEZE" which exists exclusively to be a backup of SLURP.

**Comment by Germar** [comment]
This is a known bug in 1.1.12 which was fixed in 2f11d66. But this only happens because your user has no permissions to create the folder `/media/WHEEZE/backintime`. Please check permissions.

**Comment by yourcontact** [comment]
This comment from @Germar guided me to update permissions needed so that Backintime could take snapshots: https://github.com/bit-team/backintime/issues/662#issuecomment-262071243
